buzsaki_lab_to_nwb/grosmark_code/grosmarknorecording.py

```diff
@@ -0,0 +1,26 @@
+"""Authors: Cody Baker and Ben Dichter."""
+from nwb_conversion_tools.baserecordingextractorinterface import BaseRecordingExtractorInterface
+import spikeextractors as se
+
+
+class GrosmarkNoRecording(BaseRecordingExtractorInterface):
```

Comment:
Is there something Grosmark-specific about this? Should it just be a `NoRecording`?

Reply:
It will end up being Watson, Yuta, and Grosmark specific since all the subsequent conversions will not use it. But yeah, it could, for those three, be generalized to a non-experiment specific NoRecording Interface if you wanted

Reply:
Yeah, right now this class is copied in Yuta, which isn't good. In general, I'd like to avoid copy/pasted code

Reply:
I'll go ahead and generalize it then really quick.